jetstream/config.py

```diff
@@ -421,6 +421,7 @@ class MetricsSpec:
 
     daily: List[MetricReference] = attr.Factory(list)
     weekly: List[MetricReference] = attr.Factory(list)
+    days_28: List[MetricReference] = attr.Factory(list)
```

Comment:
Python identifiers cannot start with digits. Also `_28_day` throws `invalid decimal literal`.

Reply:
heh, I forgot that `1_000_000` is a thing you can type now.